stable/prometheus-mongodb-exporter/values.yaml

```diff
@@ -0,0 +1,45 @@
+# Default values for prometheus-mongodb-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ananthunair/mongodb_exporter
```

Comment:
This should default to the official percona/mongodb_exporter image as its better maintained and community supported, not one that you maintain. Already, your branch is 19 commits behind.